lib/dal/src/cached_module.rs

```diff
@@ -132,6 +144,43 @@ impl TryFrom<PgRow> for CachedModule {
     }
 }
 
+const CACHED_MODULE_GET_FIELDS: &str = "
```

Comment:
There are an increasing number of places we reference the list of fields, so I centralized them in a const here.